README.md

```diff
@@ -1 +1,26 @@
 # Scope for Maven Action
+
+### Easy integration with Scope for Java projects based on Maven
+
+>The latest release of this Action is based on Scope Jave Agent v0.2.2
+
+## Usage
+
+Add a step to your GitHub Actions workflow YAML that uses this action:
+
+```yml
+steps:
+  - uses: actions/checkout@master
+  - name: Set up JDK 1.8
+    uses: actions/setup-java@v1
+    with:
+      java-version: 1.8 
+  - name: Scope for Maven Action
+    uses: undefinedlabs/scope-for-maven-action@v1.0.0
```

Comment:
```suggestion
    uses: undefinedlabs/scope-for-maven-action
```
Can we use it without a release? So we don't have to keep updating the README

Reply:
Based on the official docs of GitHub, to set the version is a recommended practice.

```
We strongly recommend that you include the version of the action you are using by specifying a Git ref, SHA, or Docker tag number. If you don't specify a version, it could break your workflows or cause unexpected behavior when the action owner publishes an update.

Using the commit SHA of a released action version is the safest for stability and security.
Using the specific major action version allows you to receive critical fixes and security patches while still maintaining compatibility. It also assures that your workflow should still work.
Using the master branch of an action may be convenient, but if someone releases a new major version with a breaking change, your workflow could break.
```

We could use only the major version `@v1` to allow the user receive fixes if we update our action.

Reply:
+1 to using major version